pkgs/build-support/build-dotnet-module/hooks/dotnet-configure-hook.sh

```diff
@@ -0,0 +1,32 @@
+declare -a projectFile testProjectFile dotnetRestoreFlags dotnetFlags
+
+dotnetConfigureHook() {
+    echo "Executing dotnetConfigureHook"
+
+    runHook preConfigure
+
+    if [ -z "${enableParallelBuilding-}" ]; then
+        parallelFlag="--disable-parallel"
+    fi
+
+    export HOME=$(mktemp -d)
```

Comment:
I'm not quite sure what the best place to set this is, all hooks depend on it.

Reply:
I would just export it at the earliest possible point we need it. We don't need to create a giant mess of hooks.